src/Box/types.ts

```diff
@@ -0,0 +1,28 @@
+import { BaseSpacingProps } from '../BaseSpacing';
+
+type FlexAlignType =
+  | 'stretch'
+  | 'flex-start'
+  | 'flex-end'
+  | 'center'
+  | 'baseline';
+type FlexDirectionType = 'row' | 'column';
+type FlexJustifyType =
+  | 'flex-start'
+  | 'flex-end'
+  | 'center'
+  | 'space-between'
+  | 'space-around'
+  | 'space-evenly';
+type FlexWrapType = 'nowrap' | 'wrap' | 'wrap-reverse';
+
+export interface BoxProps extends BaseSpacingProps {
+  alignItems?: FlexAlignType;
+  as?: React.ElementType | keyof JSX.IntrinsicElements;
+  children: JSX.Element | string;
+  className?: string;
+  display?: 'block' | 'flex';
```

Comment:
Do you think we should add the most common used display values?
```typescript
display?: 'block' | 'flex' | 'inline' | 'inline-block' | 'inline-flex' | 'none'
```
And if we support a Flex Box, should we consider adding props to make it easy for Flex Box Child?
```typescript
flex?: string;
flexGrow?: number | string;
flexShrink?: number | string;
// flexBasis?: 
zeroMinWidth?: boolean;
zeroMinHeight?: boolean;
```

Reply:
The first question, yes I will change it.
The second one I think we don't need a component for that and the children should use it. But we can discuss it.

Reply:
Changed!

Reply:
Yes! I meant to be Box the one having the typical props of a (Box display: flex) child. What do you think? That would allow us to use the flexbox complete resources like:
```typescript
<Box display="flex">
  <Box flexShrink="0" />
  <Box flex="1 0 auto" />
</Box>
```